pom.xml

```diff
@@ -52,28 +80,138 @@
     </dependencies>
   </dependencyManagement>
 
+  <dependencies>
+    <dependency>
+      <groupId>com.jayway.restassured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <version>2.9.0</version>
```

Comment:
Maybe move version to property?

Reply:
Good idea, I just copied this from the parent, but it's probably best we put in a property